pages/api/v1/[[...slug]].ts

```diff
@@ -30,7 +31,48 @@ router
   .get("/api/v1/projects", (req, res) => {
     res.send("this should authenticate the user ang get all projects that they can see. Yipee!");
   })
-  .get("/api/v1/projects/:id", async (req, res: NextApiResponse<Project | { error: string }>) => {
+  .patch("/api/v1/projects/:id", async (req, res: NextApiResponse<Project | { error: string }>) => {
```

Comment:
Check how we have the file path laid out here for requests that operate on items https://github.com/tigrisdata/create-tigris-app/blob/main/templates/nextjs-api-routes/pages/api/item/%5Bid%5D.ts The id can also be simply extracted from req.query